backend/src/accounts.js

```diff
@@ -63,7 +63,7 @@ async function getAccountActivity(accountId) {
   if (!accountActivity) {
     return undefined;
   }
-  const indexerCompatibilityTransactionActionKinds = await getIndexerCompatibilityTransactionActionKinds();
+  const indexerCompatibilityTransactionActionKinds = getIndexerCompatibilityTransactionActionKinds();
```

Comment:
I believe this kind of fixes should be made outside of main PR (and they can easily be merged without waiting for owner-approve)